bin/deploy.js

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env -S node --require ts-node/register
+
+const contracts = require('../src/index.ts');
+const { providers, Wallet } = require('ethers')
+const { JsonRpcProvider } = providers;
+
+const key = process.env.DEPLOYER_PRIVATE_KEY;
+const sequencerKey = process.env.SEQUENCER_PRIVATE_KEY;
+const web3Url = process.env.L1_NODE_WEB3_URL || 'http://127.0.0.1:8545';
+const MIN_TRANSACTION_GAS_LIMIT = process.env.MIN_TRANSACTION_GAS_LIMIT || 0;
+const MAX_TRANSACTION_GAS_LIMIT = process.env.MAX_TRANSACTION_GAS_LIMIT || 1000000000;
+const MAX_GAS_PER_QUEUE_PER_EPOCH = process.env.MAX_GAS_PER_QUEUE_PER_EPOCH || 1000000000000;
+const SECONDS_PER_EPOCH = process.env.SECONDS_PER_EPOCH = 600;
+let WHITELIST_OWNER = process.env.WHITELIST_OWNER;
+const WHITELIST_ALLOW_ARBITRARY_CONTRACT_DEPLOYMENT = process.env.WHITELIST_ALLOW_ARBITRARY_CONTRACT_DEPLOYMENT || false;
```

Comment:
I think this should default to `true` because most of the time we want chains to allow arbitrary contracts